xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetabledate.js

```diff
@@ -0,0 +1,86 @@
+require.config({
+    paths: {
+        moment: "$services.webjars.url('momentjs', 'moment.js')",
+        daterangepicker: "$services.webjars.url('bootstrap-daterangepicker', 'js/bootstrap-daterangepicker.js')"
+    },
+    shim: {
+        'daterangepicker': ['jquery', 'bootstrap', 'moment']
+    }
+});
+
+require(['jquery', 'moment', 'daterangepicker', 'xwiki-events-bridge'],
+function($, moment) {
+
+    // Load bootstrap-daterangepicker CSS only when needed
+    // This is a bit of a hack, but it's required until we have an API to
+    // include CSS from a webjar in Velocity.
```

Comment:
We actually have the API, see https://github.com/xwiki-contrib/application-ckeditor/blob/master/ui/src/main/resources/CKEditor/VelocityMacros.xml#L105 .